contracts/test/MockOracle.sol

```diff
@@ -159,6 +159,11 @@ contract MockOracle is ChainlinkRequestInterface, LinkTokenReceiver {
         return address(LinkToken);
     }
 
+    // @notice Validate the function called on token transfer.
+    function _validateTokenTransferAction(bytes4 funcSelector, bytes memory) internal pure override {
```

Comment:
Not sure where this has come from? in the diff view it shows as a new function but not used?